examples/pytorch/graphsage/experimental/README.md

```diff
@@ -0,0 +1,55 @@
+## Distributed training
+
+This is an example of training GraphSage in a distributed fashion. To train GraphSage, it has four steps:
+
+### Step 1: partition the graph.
+
+The example provides a script to partition some builtin graphs such as Reddit and OGB product graph.
+If we want to train GraphSage on 4 machines, we need to partition the graph into 4 parts.
+
+We need to load some function from the parent directory.
+```bash
+export PYTHONPATH=$PYTHONPATH:..
+```
+
+In this example, we partition the OGB product graph into 4 parts with Metis. The partitions are balanced with respect to
+the number of nodes, the number of edges and the number of labelled nodes.
+```bash
+# partition graph
+python3 partition_graph.py --dataset ogb-product --num_parts 4 --balance_train --balance_edges
+```
+
```

Comment:
Should we put the `--banlace_train` and `--balance_edges` as default arguments?

Reply:
sure. although this is just an example.